.ruby-version

```diff
@@ -1 +0,0 @@
-2.6.6
```

Comment:
We could leave this in here as it is only used for `development` as opposed to the `.gemspec` file one, otherwise this looks good to me 👍

Reply:
Good point @bogdanadrianmarc! I'll revert this removal to keep this file in. Thank you!